index.js

```diff
@@ -145,12 +160,12 @@ function format(answers) {
  * @type {Object}
  */
 module.exports = {
-  prompter: function (cz, commit) {
+  prompter: function(cz, commit) {
     cz.prompt.registerPrompt('autocomplete', require('inquirer-autocomplete-prompt'))
     loadConfig()
       .then(createQuestions)
       .then(cz.prompt)
-      .then(format)
+      .then(answers => format(answers, loadedConfig))
       .then(commit)
```

Comment:
We can refactor a little bit, just to avoid this `loadedConfig` global.

```js
const config = await loadConfig()

Promise.resolve()
  .then(createQuestions)
  .then(cz.prompt)
  .then(format)
  .then(answers => format(answers, config))
  .then(commit)
```